vendor-overwrites/colorpicker/colorpicker.js

```diff
@@ -552,7 +552,7 @@
   function onMouseUp(event) {
     if (releaseMouse(event, ['saturation', 'hue', 'opacity']) &&
         !event.target.closest('.codemirror-colorview, .colorpicker-popup, .CodeMirror')) {
-      hide();
+      // hide();
```

Comment:
Er... don't change it just like that. I guess it breaks some legit cases. Just add a selector into the exclusion list above. This colorpicker is ours now since it was fully rewritten so we can customize it to our needs

Reply:
Well... if we add `body` then `hide()` would never be called which is pretty much the same.
![test](https://user-images.githubusercontent.com/1324510/33628107-9bf849f6-da3a-11e7-906c-a18edafa53d8.gif)

Reply:
Can we remove it entirely?

Reply:
And do we even discuss `body`? I think you can simply add the config dialog selector. If not, then the cod should be altered so that it's possible.

Reply:
Looking at your gif, I think `hide()` should be moved to mousedown listener in colorpicker.js

Reply:
Yeah, but colorpicker doesn't register `mousedown` event on `window`. Should we add a listener for it? We can revert https://github.com/openstyles/stylus/pull/283/commits/181d9206d9fb8c98348bfa107f42a71fc85efda6 then.

Also `onMouseUp` should probably be changed to
```js
  function onMouseUp(event) {
    releaseMouse(event, ['saturation', 'hue', 'opacity']);
  }
```

Reply:
Hmm. You're right. Current colorpicker code makes no sense. I agree with your suggestions for onMouseUp and onMouseDown.